### PR TITLE
Kozak Natalia

### PR DESCRIPTION
**Pull request by Natalia1910**

### CODE REVIEW

File: src/com/company/IntegerArray.java [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    }
+
+    // Метод, який обчислює середнє значення масиву
+
+    public void averageValue(){
+        double suma = 0;
+        double average = 0;
+        for (int g : array){
+            suma += g;
+            average = suma / array.length;
```
Comment (ghost-lv) [inline src/com/company/IntegerArray.java]: you must calculate avg outside of for

File: src/com/company/IntegerArray.java [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+        System.out.print("\nEnter your array: ");
+        for (int k = 0; k < array.length; k++){
+            array[k] = s.nextInt();
+        }
+        System.out.println(Arrays.toString(array));
+    }
+
+    // Метод, який повертає суму всіх непарних індексів
+
+    public void sumAllOddIndexes(){
```
Comment (ghost-lv) [inline src/com/company/IntegerArray.java]: i think, you have to place this method in parent class

File: src/com/company/IntegerArray.java [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+        System.out.println("\nOdd indexes: ");
+        int sum = 0;
+        for (int i = 1; i < array.length; i += 2) {
```
Comment (ghost-lv) [inline src/com/company/IntegerArray.java]: 0 (first element) must be included

File: src/com/company/CharArray.java [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+              System.out.println("Yeah, it is here!");
+          } else {
+              System.out.println("Sorry, maybe next time");
+          }
+      }
+    }
+
+    // Метод, який виводить усі цифри з масиву символів
+
+    public void allNumbersInArray(){
```
Comment (ghost-lv) [inline src/com/company/CharArray.java]: task was about output of all charcodes

File: src/com/hierarchy/Telephones.java [hunk 5]
```diff
@@ -0,0 +1,76 @@
+package com.hierarchy;
```
Comment (ghost-lv) [inline src/com/hierarchy/Telephones.java]: fantastic ! ))

File: src/com/company/IntegerArray.java (hunk 2 above)
Comment (Natalia1910) [reply src/com/company/IntegerArray.java]: I announced it in my parent class and  imlemend it here!

File: src/com/company/IntegerArray.java (hunk 3 above)
Comment (Natalia1910) [reply src/com/company/IntegerArray.java]: okay, thanks

File: src/com/hierarchy/Telephones.java (hunk 5 above)
Comment (Natalia1910) [reply src/com/hierarchy/Telephones.java]: thanks)
I thought this task isn`t correct

File: src/com/company/CharArray.java (hunk 4 above)
Comment (Natalia1910) [reply src/com/company/CharArray.java]: It`s better than nothing

**Pull request merged**